src/main/java/kr/co/khacademy/semi/dao/AnnouncementDao.java

```diff
@@ -0,0 +1,96 @@
+package kr.co.khacademy.semi.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import kr.co.khacademy.semi.common.DataSource;
+import kr.co.khacademy.semi.model.Announcement;
+
+public class AnnouncementDao {
+    private static final AnnouncementDao instance = new AnnouncementDao();
+
+    private static final String INSERT_SQL = "INSERT INTO announcement VALUES (default,default,?, ?,default)";
+    private static final String SELECT_SQL = "SELECT * FROM announcement";
+    private static final String UPDATE_SQL = "UPDATE announcement SET title = ?, contents = ? WHERE id = ?";
+    private static final String DELETE_SQL = "DELETE announcement WHERE id =?";
+    private static final String SELECT_BY_ID_SQL = "SELECT * FROM announcement WHERE id = ?";
```

Comment:
아래의 메소드 순서와 맞춰주면 더 좋을것 같아요. 
그리고 update와 delete의 경우 id 조건이 들어가니 아래와 같이 수정해 주면 더 좋을 것 같아요

```suggestion
    private static final String INSERT_SQL = "INSERT INTO announcement VALUES (DEFAULT, DEFAULT, ?, ?, DEFAULT)";
    private static final String SELECT_SQL = "SELECT * FROM announcement";
    private static final String SELECT_BY_ID_SQL = "SELECT * FROM announcement WHERE id = ?";
    private static final String UPDATE_BY_ID_SQL = "UPDATE announcement SET title = ?, contents = ? WHERE id = ?";
    private static final String DELETE_BY_ID_SQL = "DELETE announcement WHERE id = ?";
```